appointment/upload-documents/uploadDocuments.php

```diff
@@ -73,21 +73,144 @@ class="submit dcf-btn dcf-btn-primary"
+						of your tax return information, your consent is valid for the amount of time that you specify. If
+						you do not specify the duration of your consent, your consent is valid for one year from the date
+						of signature. If you believe your tax return information has been disclosed or used improperly in a
+						manner unauthorized by law or without your permission, you may contact the Treasury Inspector General
+						for Tax Administration (TIGTA) by telephone at 1-800-366-4484, or by e-mail at complaints@tigta.treas.gov .
+						While the IRS is responsible for providing oversight requirements to Volunteer Income Tax Assistance (VITA) 
+						and Tax Counseling for the Elderly (TCE) programs, these sites are operated by IRS sponsored partners who
+						manage IRS site operations requirements and volunteer ethical standards. In addition, the locations of these 
+						sites may not be in or on federal property. </p>
+					<li class="form-radio" id="consentVirtual">
+						<label class="form-required">I am agreeing to use this site's Virtual VITA/TCE Process</label>
+						<div>
+							<div class="dcf-btn-group" data-toggle="buttons">
+								<label class="dcf-btn dcf-btn-secondary" 
+									name="consentToVirtual" 
+									ng-model="consentData.virtualConsent" 
+									uib-btn-radio="true" 
+									required>Yes
+								</label>
+								<label class="dcf-btn dcf-btn-secondary" 
+									name="consentToVirtual" 
+									ng-model="consentData.virtualConsent" 
+									uib-btn-radio="false" 
+									required>No
+								</label>
+							</div>
+						</div>
+						<div ng-show="consentForm.$submitted">
+							<label class="error" ng-show="consentForm.consentToVirtual.$error.required">This field is required.</label>
+						</div>
+						<p class="cant-help-text" ng-show="consentData.virtualConsent == false">Sorry, we cannot prepare your tax return without your consent to the virtual process.</p>
+					</li>
+
+					<li class="dcf-form-group form-textfield">
+						<label class="dcf-label form-label form-required" for="signature">TYPE YOUR NAME HERE</label>
+						<input type="text" class="dcf-input-text form-control" name="signature" id="signature" ng-model="consentData.signature" required>
+						<div ng-show="consentForm.$submitted || consentForm.signature.$touched">
+							<label class="error" ng-show="consentForm.signature.$error.required">This field is required.</label>
+						</div>
+					</li>
+
+					<p ng-if="consentForm.$invalid && consentForm.$submitted" class="error">Your input is invalid, please correct the errors above and re-submit this form</p>
+					<input type="submit" 
+						value="Submit" 
+						class="submit dcf-btn dcf-btn-primary dcf-mt-4" 
+						ng-disabled="!consentData.virtualConsent == true || consentData.reviewConsent == null || consentData.signature == null || consentData.signature == ''" />
```

Comment:
This should be `consentData.virtualConsent != true || .... || consentData.signature.trim() == ''`